### ISSUE TITLE
PR: Revit 2024 User Property Set not exported

### ISSUE THREAD

**Issue by woookieee** [Autodesk/revit-ifc#717]
### Problem Description

I'm trying to export some of Revit parameters translated through User Property Set so they appear in new tab in IFC viewing software (solibri). I'm using custom User Property Set file (according to table below). 
![image](https://github.com/Autodesk/revit-ifc/assets/152966162/c872704d-cd61-42ab-9257-68b5044b5a52)

the issue i encounter is that I get the new tab in Solibri called BIP, however none of the specified parameters are translated and showing up in the tab. 
![image](https://github.com/Autodesk/revit-ifc/assets/152966162/a0737c08-cede-4da8-bec1-fd2a3ac7ab5b)

I'm aware of similar issue in Revit 2023 which was resolved by installing an older version of IFC addon. This however doesn't seem to work in Revit 2024.

Do any of you have any idea how to resolve this issue? Or is it some kind of bug in the IFC version? 
all the best, 
Lukas 

### Revit Version

2024.0.x

### IFC for Revit Addon Version

24.x.x

### Windows Version

11 22H2

**Comment by Nephel66** [comment]
Hi ! Unfortunately, there is the same issue with IFC exporter V23.3.0 et V23.3.1.

**Comment by IbimDesarrollo** [comment]
I am experiencing same problems with properties that came from types, like:
![image](https://github.com/Autodesk/revit-ifc/assets/93908242/ae2a9d58-eed9-43b5-8dac-4b58f243bf69)

With older version like RVT2020 works, but in RVT2023 it already fails.

**Comment by woookieee** [comment]
@IbimDesarrollo Yes, in the previous versions of Revit (e.g. 2023) the workaround for this issue was to downgrade IFC-addon to earlier version. Unfortunately, I don't find this working in Revit 2024, I really have no idea how to approach this in this case.

**Comment by andydandy74** [comment]
I think you may have to differentiate between instance and type parameters in 2024: https://bim-me-up.com/en/revit-typenparameter-nach-ifc-exportieren/

**Comment by eibre** [comment]
Changing I to T in the txt-file helped for me (in Revit 2024), even though the parameters was set on instance. Looks like a bug?

**Comment by AngelVelezSosa** [comment]
It's as intended but it is confusing, so we intend to do something to make it easier to use old mapping files.

**Comment by peterlund0** [comment]
I can't in any way create a Pset with Type Parameters, not even when all Psets are set to "T". It is version 24.2.0.0. Is this planned to be resolved any time soon?

**Comment by lejla-adsk** [comment]
T is obsolete, use IfcElementType in the configuration file (or the corresponding Ifc*Type for your classes). 
For Doors and WIndows in IFC2x3, use IfcDoorStyle / IfcWindowStyle.

**Comment by peterlund0** [comment]
Thanks! thar seemed to do the trick!
